README.md

```diff
@@ -22,148 +19,183 @@ Table of contents
-6. For information on how to populate this sample database, see the developer documentation [Populate the Database](https://developers.elasticpath.com/commerce/7.3/Core-Commerce-Development/Setting-up-your-Developer-Environment/Populate-the-Database#ConfigureDemoDataSets).
-
-### Setup (Development):
-1. Clone or pull this repository to a directory of your choosing.
-2. Run `cd ep-store`.
-3. Run `npm install` to install any dependencies.
-4. Configure the `./src/ep.config.json` file as required for your environment: [here](#configuration)
-5. Run `npm start` to start the server in development node.
-6. Navigate to `http://localhost:8080/` to see the running Progressive Web Application (PWA).
-
-### Setup (Production):
-1. Clone or pull this repository to a directory of your choosing.
-2. Run `cd ep-store`.
-3. Run `docker build -t ep-store -f ./docker/prod/Dockerfile .`
-4. Push `ep-store` image to your docker repository.
-
-##### On a host computer:
-5. Pull `ep-store` from your docker repository.
-6. In the repository, navigate to `docker/prod/`. Copy the provided `docker-compose.yaml` and `nginx.conf` files to a folder on the remote host.
-7. Replace `$CORTEX_URL` in `nginx.conf` with a URL of your cortex server as well as `$DOCKER_REPO` in `docker-compose.yaml` with `ep-store`.
-8. Run `docker-compose up -d`.
-
-## Linting
-The project has been set up with ESLint as our linting utility.<br/>
-For more infoarmtion about ESLint, see [ESLint documentation](https://eslint.org/).<br/>
-We're currently extending Airbnbs’ ESLint configuration. Currently, the Airbnb Code Style and the ESLint configuration are very popular and well accepted by developers.<br/>
-For more infomation on the style guide, see the [Airbnb GitHub](https://github.com/airbnb/javascript).<br/>
-* The ESLint loader is already added to`webpack.config.dev.js`. When you start the application in development mode, the ESLint loader automatically runs.
-* To run the linter from your commandline, go to the project root directory. Run the following command:
-    * `./node_modules/.bin/eslint --ext .js --ext .jsx [file|dir|glob]`
-        * Run on entire project: `./node_modules/.bin/eslint --ext .js --ext .jsx .`
-* With the ESLint extension for Visual Studio Code, you'll get feedback while you're writing the code under the `Problems` View.
-
-If you plan to check in your code, ensure that you fix all of your linting errors first.
+5. To run the Data Population tool, run the following command:<br/> `mvn clean install -Preset-db`.
+For more information about populating database, see the [Populating the Database](https://developers.elasticpath.com/commerce/7.3/Core-Commerce-Development/Setting-up-your-Developer-Environment/Populate-the-Database#ConfigureDemoDataSets) section.
+
+### Setting up the Development Environment
+
+1. Clone or pull the `react-pwa-reference-storefront` repository to your directory.
+2. Run the `cd ep-store` command.
```

Comment:
I just noticed this is actually incorrect/outdated! Since we moved the repository, the command should now be `cd react-pwa-reference-storefront` as developers are just navigating into the directory. I don't feel we need to give the impression it's a custom command either as they're just navigating.